pom.xml

```diff
@@ -131,7 +131,7 @@
         <jetty.version>9.4.48.v20220622</jetty.version>
         <javax-servlet.version>3.1.0</javax-servlet.version>
         <strimzi-oauth.version>0.11.0</strimzi-oauth.version>
-        <netty.version>4.1.86.Final</netty.version>
+        <netty.version>4.1.89.Final</netty.version>
```

Comment:
In the bridge I decided to set 4.1.87 because that's the version that Vert.x 4.38 brings to us in order to avoid having some Netty modules 4.1.87 (because of Vert.x) and other Netty modules on a different version (i.e. 4.1.89 in your case). Maybe it's better to stick with what Vert.x is bringing? @scholzj wdyt? Otherwise I will have to bump Netty on bridge as well to align with the operator at least.

Reply:
I think sticking with what VErt.x uses is a good idea.

Reply:
Okay, I will revert it to 4.1.87